src/main/java/org/jboss/xavier/analytics/pojo/output/WorkloadInventoryReportModel.java

```diff
@@ -51,13 +60,28 @@
     private Integer memory;
     private Integer cpuCores;
     @ElementCollection(fetch = FetchType.EAGER)
+    @CollectionTable(
+            indexes = {
+                    @Index(columnList = "workload_inventory_report_model_id", unique = false)
+            }
+    )
     private Set<String> workloads;
     private String complexity;
     // with "IMS" suffix in case one day we will have
     // their "AMM" counterparts
     @ElementCollection(fetch = FetchType.EAGER)
+    @CollectionTable(
+            indexes = {
+                    @Index(columnList = "workload_inventory_report_model_id", unique = false)
```

Comment:
`unique` value is by default `false` so no need to write it again.

Reply:
As above